src/test/kotlin/blackjack/DeckTest.kt

```diff
@@ -32,4 +32,13 @@ class DeckTest {
 
         assertThat(deck.cards.contains(card)).isFalse()
     }
+
+    @Test
+    fun `Generated new cards if one card left`() {
+        val deck = Deck()
+        deck.drawCards(51)
+        deck.drawCards()
+
+        assertThat(deck.cards.size).isEqualTo(51)
+    }
```

Comment:
This is just a question about the way game works:

If there's only one card left, does that card get discarded, and new set of deck get created?